components/media-feature/Component.jsx

```diff
@@ -43,16 +43,19 @@ export default function MediaFeature({
               <FeaturedHeading type={mediaType} />
             </div>
 
-            <a href={linkUrl} className="su-no-underline">
-              <h3 className="su-text-[35px] su-font-bold su-leading-tight su-m-0 su-pb-8 su-text-black md:su-pb-19 md:su-text-[40px] lg:su-text-[43px]">
+            <h3 className="su-text-[3.5rem] su-font-bold su-leading-tight su-m-0 su-pb-8 su-text-black md:su-pb-19 md:su-text-[4rem] lg:su-text-[4.3rem]">
+              <a
+                href={linkUrl}
+                className="su-group su-no-underline hocus:su-underline su-stretched-link su-text-black hocus:su-text-digital-red dark:su-text-black dark:hocus:su-text-digital-red"
+              >
                 {title}
-                <span className="su-hidden lg:su-inline-block su-relative su-top-12">
-                  <ExternalArrow size="large" />
+                <span className="su-hidden lg:su-inline-block su-relative su-top-12 group-hocus:su-translate-x-01em group-hocus:su--translate-y-01em su-transition-transform">
+                  <ExternalArrow size="large" title="link is external" />
                 </span>
-              </h3>
-            </a>
+              </a>
+            </h3>
```

Comment:
I changed the structure of the Media Feature card a bit. I believe it's better to have heading around a link a tag than the other way around. Also, making the whole card clickable using su-stretched-link class in the link element

Reply:
Also added a title "link is external" to the external arrow icon for accessibility